apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt

```diff
@@ -28,12 +28,9 @@ open class GraphqlCompiler {
 
   private fun String.formatPackageName(): String {
     val parts = split(File.separatorChar)
-
-    for (i in 2..parts.size) {
-      if (parts[i - 2] == "src" && parts[i] == "graphql") {
-        return parts.subList(i + 1, parts.size).dropLast(1).joinToString(".")
-      }
-    }
+    (2..parts.size)
```

Comment:
What do you think about `parts.size - 1 downTo 2` instead? 
It doesn't really matter but that should handle all the cases now 😀 (including the `src/foo/grahpql/src/bar/graphql/Query.json`)